daemon/containerd/image_list.go

```diff
@@ -276,10 +279,15 @@ func (i *ImageService) singlePlatformImage(ctx context.Context, contentStore con
 		return nil, nil, err
 	}
 
+	created := int64(0)
```

Comment:
Maybe we could leave the `rawImg.CreatedAt.Unix()` as a fallback in case the `cfg.Created` is nil?

Reply:
I'm sure @thaJeztah can link you some issue here or in the cli about what should we do with the images that don't have a creation date. It's a real thing, especially for reproducible builds so I think we should just set `0` if there is no creation date

Reply:
There was some discussion around dates and reproducibility in https://github.com/moby/moby/pull/45504#discussion_r1189110803 (not sure if relevant in any way)